README.md

```diff
@@ -1,11 +1,49 @@
 
-A new Flutter project.
+A Flutter project.
 
-## Getting Started
+## OBJECTIVE:
+This project for Data Day Grind 3.0:  with a aims to develop a Tic Tac Toe game using Flutter. It mainly consists of developing a Tic Tac Toe game.
+In order to understand what Tic Tac Toe game is and how to play the game, below is the description.
 
-This project is a starting point for a Flutter application.
+### GAME DESCRIPTION:
+Tic Tac Toe is a two-player game. In this game, there is a board with 3 x 3 squares.
 
+The two players take turns putting marks on a 3x3 board. The goal of Tic Tac Toe game is to be one of the players to get three same symbols in a row - horizontally, vertically or diagonally on a 3 x 3 grid. The player who first gets 3 of his/her symbols (marks) in a row - vertically, horizontally, or diagonally wins the game, and the other loses the game. The game can be played by two players. 
+
+### GAME RULES:
+A player can choose between two symbols with his opponent, usual game uses “X” and “O”.
+
+The player that gets to play first will get the "X" mark (we call him/her player 1) and the player that gets to play second will get the "O" mark (we call him/her player 2).
+
+Player 1 and 2 take turns making moves with Player 1 playing mark “X” and Player 2 playing mark “O”.
+
+A player marks any of the 3x3 squares with his mark (“X” or “O”) and their aim is to create a straight line horizontally, vertically or diagonally with two intensions:
+a. One of the players gets three of his/her marks in a row (vertically, horizontally, or diagonally) i.e. that player wins the game.
+b. If no one can create a straight line with their own mark and all the positions on the board are occupied, then the game ends in a draw/tie.
+
+### UI:
+
+![1 (3)](https://user-images.githubusercontent.com/65271275/180607342-a39a1de2-ee9e-40b9-9692-ef3132590250.jpg)
+![2](https://user-images.githubusercontent.com/65271275/180607378-e29b9e4a-1fb5-4312-9ec9-53a304ab2ebb.jpg)
+![3](https://user-images.githubusercontent.com/65271275/180607382-a5573e48-58cf-4bce-832f-281e7cc12cc4.jpg)
+![4](https://user-images.githubusercontent.com/65271275/180607384-1a4466fe-d45a-424d-a621-dd6155757940.jpg)
+![5](https://user-images.githubusercontent.com/65271275/180607386-77a9f32c-60b8-4198-9233-5fc93d470fd6.jpg)
+
+
+# Contribution Guidelines
+
+## Steps to contribute to this project
+
+- Make sure to fork this repo using the `fork` button on the top
+- Then clone it via git `git clone [https://github.com/<your-username>/Tic-Tac-Toe-Flutter](https://github.com/hasnainmakada-99/Tic-Tac-Toe-Flutter)`
+- Open it in VS Code or Android Studio
+- Install all the dependencies using the `flutter pub get` command
+- Plug in your physical device or connect your emulator, Make sure to enable USB Debugging
+- Run the application using `flutter run` command
+- And Your application is up and running. Enjoy 😃
+
+# Resources Link 
 A few resources to get you started if this is your first Flutter project:
```

Comment:
Why are this coming again and again, Have you not pulled the changes in your forked repo

Reply:
I do not about this, I am also figuring out this